tests/tests_unit/test_api/test_sequences.py

```diff
@@ -135,7 +135,7 @@ def mock_get_sequence_data_two_col(rsps, cognite_client):
     }
     rsps.add(
         rsps.POST,
-        cognite_client.sequences._get_base_url_with_base_path() + "/sequences/data/list",
+        f"{cognite_client.sequences._get_base_url_with_base_path()}/sequences/data/list",
```

Comment:
I do think I prefer the original here when the variable has such a long dotted name.